src/test/java/seedu/address/logic/parser/CatalogParserTest.java

```diff
@@ -50,9 +50,9 @@ public void parseCommand_clear() throws Exception {
 
     @Test
     public void parseCommand_delete() throws Exception {
-        DeleteCommand command = (DeleteCommand) parser.parseCommand(
-                DeleteCommand.COMMAND_WORD + " " + INDEX_FIRST_BOOK.getOneBased());
-        assertEquals(new DeleteCommand(INDEX_FIRST_BOOK), command);
+        DeleteByIndexCommand command = (DeleteByIndexCommand) parser.parseCommand(
+                DeleteByIndexCommand.COMMAND_WORD + " " + INDEX_FIRST_BOOK.getOneBased());
+        assertEquals(new DeleteByIndexCommand(INDEX_FIRST_BOOK), command);
```

Comment:
I think that this needs to be updated as the delete command now contains two subclasses which are deleteByIndex  and deleteBySerialNumber.

Reply:
Test will be included in next commit